apps/easypid/src/features/wallet/FunkeOpenIdPresentationNotificationScreen.tsx

```diff
@@ -41,11 +42,21 @@ export function FunkeOpenIdPresentationNotificationScreen() {
   }, [router.back])
 
   useEffect(() => {
-    getCredentialsForProofRequest({
-      agent,
-      data: params.data,
-      uri: params.uri,
-    })
+    ;(async () => {
+      // TODO: we have to detect whether the verifier requests something from the b' flow (maybe check iss field in the pres def?)
+      await requestSdJwtVcFromSeedCredential({
+        agent,
+        authorizationRequestUri: params.uri ?? 'TODO: this is temp anyways',
+        pidPin: '8376487',
+      })
+    })()
+      .then(() =>
+        getCredentialsForProofRequest({
+          agent,
+          data: params.data,
+          uri: params.uri,
+        })
+      )
```

Comment:
weird formatting.

```suggestion
      requestSdJwtVcFromSeedCredential({
        agent,
        authorizationRequestUri: params.uri ?? 'TODO: this is temp anyways',
        pidPin: '8376487',
      })
      .then(() =>
        getCredentialsForProofRequest({
          agent,
          data: params.data,
          uri: params.uri,
        })
      )
```

Also re comment: `// TODO: we have to detect whether the verifier requests something from the b' flow (maybe check iss field in the pres def?)`

I think we can detect it based on whether the eph_key is included in the request (we should only include it if requesting b' flow on the RP side)

Reply:
Yeah that works